### ISSUE TITLE
Use this repo as a base for the official Chinese React docs

### ISSUE THREAD

**Issue by tesseralis** [discountry/react#334]
Hi! I'm working on the official translations for the React docs and @gaearon and I are impressed with the quality of this repo. We're actually thinking of doing this approach more generally (see: https://github.com/reactjs/reactjs.org/issues/1605) and we'd like to use your repo as a starting point. Would you like to work with us into making this the official Chinese React documentation?

**Comment by discountry** [comment]
That sounds great, how should we proceed?

**Comment by tesseralis** [comment]
Thanks for replying!

For now, I can think of two things:

* Sync this fork with the original repository (reactjs/reactjs.org), making sure that logic and formatting are consistent (I can help with this).
* Ensure that the following sections are translated and up-to-date with the current React docs:
  * Home page
  * Tutorial
  * Main Concepts
  * API

There are other issues (in particular, making sure that [anchor names for headings are consistent](https://github.com/reactjs/reactjs.org/issues/1605#issuecomment-458816106), but those can wait until changes to the main reactjs.org repository.

Again, thank you so much for the help!

**Comment by gaearon** [comment]
I was thinking that maybe it would be a good idea to start a fresh fork from current English master, and apply translations on top on a file-by-file basis (while checking that they match the content). If some pages have changed significantly, leave them in English so we can come back to them in another pass.

**Comment by tesseralis** [comment]
As @gaearon mentioned, I think we'd rather start with a new fork instead (Dan already set one up at https://github.com/reactjs/cn.reactjs.org).

Doing this has the advantage of:

* Allowing us to cherry-pick commits to make sure everything is up to date.
* Making sure we don't lose recent website changes.
* Making sure that translations don't "roll back", i.e. we don't replace "Hello World" with a translated version of it from two years ago.

**Comment by lcxfs1991** [comment]
@gaearon @tesseralis 

Hi I am Lee from Tencent. A few friends and I had a side project [docschina](https://github.com/docschina) to help localize tech docs. We have run our docs translation team for two years. We have help webpack, rollup, parcel to build their Chinese docs. From our experience, it is hard to keep the docs up to date and it is sometimes hard for Chinese developers to reach the site because github and github pages are usually not stable in China. 

My suggestion is that we can use part of the translation content from this repo and start a new one.
We have created one: [https://github.com/docschina/cn.reactjs.org](https://github.com/docschina/cn.reactjs.org). Then we should also deploy a localized copy purely for Chinese developers like webpack, rollup and etc.

**Comment by lcxfs1991** [comment]
@gaearon @tesseralis 

We have this Chinese official docs for a long time.

https://react.docschina.org/

**Comment by discountry** [comment]
The problem is that our contributors have synced the content between this repo and the official one manually, but the rest files maybe out-dated. And there are also *hard-coded content* in the 'src' website templates.

I think we should start with the newest official repo, and try to diff the contents file by file.

As @lcxfs1991 mentioned, we've set up a Chinese version doc site for a long time, maybe we can just resolve the `cn.reactjs.org` domain to it and work with the updates at the same time.

**Comment by gaearon** [comment]
I don’t understand which parts of the proposal you’d like to do differently. Can you explain in more detail?

We thought it would make most sense to:

* Keep the fork in ReactJS GitHub org
* You would be owners of that fork
* You would move content from this repo into that fork page by page, verifying it’s up-to-date
* We would set up a bot that would help you stay up to date by sending PRs and issues 

Which parts do you disagree with? We could do it differently. But I’m not sure I understand what it is that you propose to change. Thanks!

**Comment by gaearon** [comment]
I don’t mind making a repository in another org (e.g. docschina) the official one instead of one in our org (reactjs). Can you explain why this is better?

My concerns are:

* I want website source to match our latest master
* I want to make sure translated docs match current content and aren’t old

I thought it would be easier to enforce if we move everything into our repo page by page. You will have full control over the repo itself.

But if you want to keep it in docschina org that’s also fine. Please let me know what works better for you.

**Comment by discountry** [comment]
@gaearon There is no disagreement.

We can collaborate under  [https://github.com/reactjs/cn.reactjs.org](https://github.com/reactjs/cn.reactjs.org).

The bot and auto-sync with official doc would also be helpful.

We'll try to merge the content of this repo to the newest version.

And we already have our translator team, once we got the authority it would be helpful for us to add people to collaborate.

**Comment by gaearon** [comment]
Amazing. If there’s anything we can help you with, let us know!

**Comment by gaearon** [comment]
@discountry Who should I add to the team?

**Comment by gaearon** [comment]
Looks like work started in https://github.com/reactjs/cn.reactjs.org/pull/3

**Comment by QC-L** [comment]
@gaearon Yeah, you should add @discountry @lcxfs1991 @hijiangtao @dear-lizhihua

**Comment by QC-L** [comment]
@gaearon Oh, I think the issue of organizing people should be controlled by us. There shouldn’t be too many people, otherwise it will be inconvenient.

**Comment by gaearon** [comment]
I have sent the invitations, you should have access now after you accept them.

**Comment by QC-L** [comment]
@gaearon Oh, that’s great. THX!
